js/dropdowns.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+/* global require, module, document */
+
+var $ = require('jquery');
+var perfectScrollbar = require('perfect-scrollbar/jquery') ($);
+var KEYCODE_ESC = 27;
+
+/**
+ * Dropdown toggles
+ * @constructor
+ * @param {string} mainSelector - CSS selector for the fieldset that contains everything
+ */
+
+function Dropdown(mainSelector) {
+    var self = this;
+
+    self.isOpen = false;
+    self.hasPanel = true;
+
+    self.$body = $(mainSelector);
+    self.$selected = this.$body.find('.dropdown__selected');
+    self.$button = this.$body.find('.dropdown__button');
+    self.$panel = this.$body.find('.dropdown__panel');
+    self.$checkboxes = this.$panel.find('input[type="checkbox"]');
+
+    self.$button.on('click', this.toggle.bind(this));
+    self.$checkboxes.each(function(){
```

Comment:
Instead of looping over checkboxes and binding a separate listener to each one, this might be a good use case for event delegation:
```javascript
self.$panel.on('change', 'input[type="checkbox"]', this.toggle.bind(this));
```
That way, we only have to use one listener, and checkboxes that move outside the panel no longer trigger the callback. Docs at http://learn.jquery.com/events/event-delegation/.